airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -199,13 +200,32 @@ def clear_not_launched_queued_tasks(self, session: Session = NEW_SESSION) -> Non
             assert self.kube_client
         from airflow.models.taskinstance import TaskInstance
 
+        has_executor = hasattr(TaskInstance, "executor")
+        default_executor = None
+        if has_executor:
```

Comment:
`has_executor` is being used here as a mechanism to check if multiple executor configuration is available right? For backcompat?

If so, I'd rename this variable because it makes the code hard to grok if you don't  have the details and there are no comments.

Maybe `hybrid_executor_enabled`?